improver_tests/utilities/test_OccurrenceWithinVicinity.py

```diff
@@ -82,17 +82,14 @@ def cube_fixture() -> Cube:
     )
 
 
-DISTANCE = 2000
+RADIUS = 2000
+GRID_POINT_RADIUS = 1
```

Comment:
I think there ought to be a test for these items being zero (as opposed to the default `None`). The behaviour should be the same as `None`.

Reply:
I've modified the plugin to handle being given a distance of 0 or None, or a grid point radius of None; it was already fine with a grid point radius being set as 0. I've added unit tests to cover this and demonstrate the input data is returned unchanged.